ui/src/components/IconWithTitle/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import classnames from 'classnames';
+import Icon from 'components/Icon';
+
+import './icon-with-title.scss';
+
```

Comment:
This can be simplified, only used once and most styling and functionality here is not needed.